spec/support/fill_in_the_form_steps.rb

```diff
@@ -127,7 +127,7 @@ def and_is_worried_about_mental_health
   def and_is_not_able_to_leave_home_if_absolutely_necessary
     expect(page).to have_content(I18n.t("coronavirus_form.groups.leave_home.questions.able_to_leave.title"))
 
-    choose "I cannot go out because I have coronavirus, or someone in my household does"
+    choose "I should not leave home because I have coronavirus symptoms, or someone in my household does"
```

Comment:
Do you think it would be easier if we just pulled in the content directly from the I18n file, like we do for the expectation directly above?

Reply:
Yes, absolutely. But that's a bigger refactor to everything that we should write a separate card for.

Reply:
I'll add a card

Reply:
https://trello.com/c/aZdkhCC0